lib/web3-utils.js

```diff
@@ -53,3 +53,18 @@ exports.waitForTransaction = function(tx, gasLimit) {
   });
 
 }
+
+exports.toBigNumberWithDecimals = (number, decimalPlaces=0) => {
+  if (!Number.isInteger(decimalPlaces) || decimalPlaces < 0) {
+    throw new TypeError("decimalPlaces should be non negative integer");
+  }
+  var bignum = new web3.BigNumber(number).times(new web3.BigNumber(10).pow(decimalPlaces));
+  if (!bignum.isInteger())
+    throw new RangeError("Precision higher then " + decimalPlaces + " decimal places");
```

Comment:
`Conversion to BigNumber requires higher precision then ${decimal places} decimal places.`